proposals/stage-0/stage-1/community-board-representation/README.md

```diff
@@ -0,0 +1,64 @@
+# Community Board Representation
+>  Stage 1
+
+## Champion
+
+[Myles Borins](https://github.com/MylesBorins)
+
+## Description
+
+### What exists today
+
+Currently the Node.js foundation has up to 3 community board seats:
+
+* 1 board seat for TSC
+* 1 board seat for Community Committee
+* 1 board seat for Individual Membership Program
+
+There are some limitations on these seats as noted in [Section 4.3](https://github.com/nodejs/board/blob/master/by-laws.md#section-43-nomination-election-and-term-of-office-of-directors),
+specifically that:
+
+4.3 d) There must be 3 platinum directors for the individual membership theo elect a board member
+
+4.3 e) There must be 2 platinum directors for the TSC or CommComm to elect a board member
+
+### Proposal for $FOUNDATION
+
+There would be up to 3 community board seats:
+
+* 1 board seat for CPC
+* 1 floating board seat to be delegated by CPC
```

Comment:
The challenge I see here is how we address the discussion around representation for the Node.js project.  With 2 it is easier to have a split which addresses the issue as outlined in: https://github.com/nodejs/bootstrap/pull/70. Any thoughts of how we address the issue if there is only one seat.  We could just have a vote from all members and since Node.js has more members then it would be able to select a representative.  Downside is that other projects may not feel fully represented. On the other hand it does not seem fair to set up an approach for other smaller projects with less total members could select the representative over that the Node.js community might want.

The other advantage is that if for some reason the individual membership program does not workout or grow, the CPC would have the flexibility to chose some other process for the second seat.

Reply:
The vote would be only by voting members of CPC afaik... meaning node.js would be only 2 votes of arguably between 6 - 12 votes. This would be the case for anything that the CPC would vote on.

Re: floating seats vs non floating. If the membership program doesn't work we can always adjust bylaws.. this is something we did re: commcomm. I think that having the seat be activated by number of members will be a great motivation for the projects in the foundation to help the program be successful

Reply:
I don't think that is the way it works for CNCF, its a vote among all of the members.  If it is the only CPC members then we are back to the discussion about Board representation for the Node.js project (either confirming that there is/is not an expectation on that front as has been expressed and if there is how to respond to that).

Reply:
There are two different stages here. The first would be the CPC decided how the seat would be allocated (and likely process for allocation. Then following through on that process.

The CPC could decide to make the seat voted on by the entire membership, they could also decide to give the seat directly to the Node.js project to design a process around, or alternatively could assign it to a specific person.

Reply:
My assumption of this proposal was that Node.js is separate. We haven't solidified or separated the idea of whether all constellation projects get board seats or JUST Node.js, but I think that the Node.js seat is very separate from this overall.

Reply:
@MylesBorins I agree with what you say, but when I said
> We could just have a vote from all members and since Node.js has more members then it would be able to select a representative. 

I meant all members of the combined projects within the Foundation, as that was how the process was outlined for voting in the initial proposal.

Reply:
@mhdawson when I had initially proposed the floating board seat I never intended for the allocation to be voted on by all members. Can you point me to the proposal where that was outlined

Reply:
to me, this could be an interesting way for projects to rotate representation on the board, as their needs and priorities change. If there were a particularly short term limit for this seat, say a year, I could see the seat going to Node for a term or two, then say NodeRed or JerryScript for a term, etc as needed. It would require that these projects are actively participating in the CPC and communicating with their peers about what they're doing. IMO direct access to the board for each project shouldn't need to be a constant thing - just a need when say, the project is trying to achieve greater adoption or corp support

Reply:
The size of a project will make it more likely that a board member will be elected from that project but it doesn't guarantee it. Project members can and should vote for a person who can represent their interests, whether they originate in their project or not. Given the scope of the seat's responsibilities, a person should ideally have some exposure beyond their own project.

On the one hand, a board member from Node.js project could make the other projects have too little direct representation. On the other hand, a board member from a small project could have outsize direct representation.

Having the seat be activated by number of members seems a good way to give projects the right incentives.